.travis.yml

```diff
@@ -14,6 +14,7 @@ install:
   - conda create -n test-environment python=$TRAVIS_PYTHON_VERSION pandas numpy
   - source activate test-environment
   - conda install --file conda-requirements.txt # xlrd, possibly others
+  - conda install -c ospc taxcalc
```

Comment:
@PeterDSteinberg - Does this look ok?

FYI - the taxcalc version on conda is not yet able to handle what B-Tax does.  Those changes have only recently been incorporated into OSE/Tax-Calculator:master